src/svg/SvgExport.js

```diff
@@ -253,6 +253,18 @@ new function() {
     }
 
     function exportText(item) {
+        // incase text is multiline create a group
+        if (item._lines.length > 1) {
+			var nodeGroup = SvgElement.create('g');
+			for (var i in item._lines) {
```

Comment:
Could you change this to a index based loop? We don't use `in` for arrays in paper.js:
```js
var lines = item._lines;
for (var i = 0, l = lines.length; i < l; i++) {
```